lib/xdrgen/generators/rust/src/types.rs

```diff
@@ -1,5 +1,8 @@
 use core::{array::TryFromSliceError, fmt, fmt::Debug, ops::Deref};
 
+#[cfg(feature = "serde")]
+use serde::{Deserialize, Serialize};
```

Comment:
I'd be inclined to remove the `use` and fully qualify the serde types at their call sites? I don't think it will be too burdensome, and I think it will aid readability since the type names are rather generic and the crate name will help provide context at their use.

Reply:
Done